example/simple/main.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// The simple command demonstrates a simple functionality which
+// prompts the user for a GitHub username and lists all the public
+// organization memberships of the specified username.
+
```

Comment:
This package comment is not formatted correctly and won't show up in `godoc`.

The package comment needs to be directly above the `package` clause, without a blank line in between. It should also be separate from the license comments above.

See https://golang.org/doc/effective_go.html#commentary for more information about the package comment. See [here](https://github.com/google/go-github/blob/0c3b302de2a6de84a2511db47ea1bb2ff8146830/example/basicauth/main.go#L1-L9) for another example that does this right.

Reply:
Got it! Thanks.